transcription/transcription.go

```diff
@@ -29,3 +30,15 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// ConvertAudioIntoRequiredFormat converts encoded audio into the required format.
+func ConvertAudioIntoRequiredFormat(fn string) error {
```

Comment:
How are you running the tests?

On Sat, Mar 12, 2016, 10:57 AM Yoni Nachmany <notifications@github.com>
wrote:

> In transcription/transcription.go
> <https://github.com/hack4impact/audio-transcription-service/pull/19#discussion_r55923069>
> :
>
> > @@ -29,3 +30,15 @@ func msgHeaders(from string, to []string, subject string) string {
> >  	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
> >  	return strings.Join(msgHeaders, "\r\n")
> >  }
> > +
> > +// ConvertAudioIntoRequiredFormat converts encoded audio into the required format.
> > +func ConvertAudioIntoRequiredFormat(fn string) error {
>
> Tried resolving this problem for some time, not sure how to: [image:
> screen shot 2016-03-12 at 10 54 40 am]
> <https://cloud.githubusercontent.com/assets/4250521/13723903/32cc0568-e841-11e5-9edc-5b5b5b712d4c.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hack4impact/audio-transcription-service/pull/19/files#r55923069>
> .
>